StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/AccessoryButton.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  AccessoryButton.swift
+//  StripePaymentSheet
+//
+//  Created by Nick Porter on 5/28/24.
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+@_spi(STP) import StripeUICore
+import UIKit
+
+final class AccessoryButton: UIButton {
+
+    enum AccessoryType {
+        case none
```

Comment:
This doesn't seem right, we should remove it. The `none` case isn't a type of AccessoryButton - you can't initialize one with this value.

Reply:
Fair point, removed this.